actionpack/lib/action_dispatch/routing/mapper.rb

```diff
@@ -1935,6 +1935,11 @@ def path_scope(path)
           end
 
           def map_match(paths, options)
+            ActionDispatch.deprecator.warn(<<-MSG.squish) if paths.count > 1
```

Comment:
For longer lists where duplicating the `:to` option is too burdensome, there is an opportunity to encourage something like [Object#with_options](https://edgeapi.rubyonrails.org/classes/Object.html#method-i-with_options):

```ruby
with_options to: "users#index" do 
  get "/users"
  get "/other_path"
  # ...
end
```

If that feels too complex or a solution, there is an opportunity to encourage looping:

```ruby
["/users/", "/other_path"].each { |path| get path }
```

Is the deprecation message an appropriate place to suggest code alternatives, or would that too presumptive and noisy?

Reply:
I left it at `Please use multiple method calls instead` because in this codepath, the developer could be using `draw`, `get`, or some other metaprogramming. I can drop these hints in the changelog entry instead.